CMSgov/easi-app
# [NOREF] IT Gov v2 cleanup

# NOREF

## Changes and Description

- Updates from [Slack thread with Zoe and Jeff](https://cmsgov.slack.com/archives/CNU2B59UH/p1697041581265419):
  - Added `trbFollowup` field to Life Cycle ID admin page
  - Replaced all instances of “Lifecycle ID” with “Life Cycle ID”
- Fixed misaligned Business Case PDF export button
    - Button was displayed outside of `grid-container`
- Minor fixes to spacing and translation text in admin views



## How to test this change
- Log in as IT Gov admin with `itGovV2Enabled` flag turned on
- Run `scripts/dev db:seed`
- `trbFollowup` field
  - Issue LCID to intake
  - Check Life Cycle ID admin page for "Should this team consult with the Technical Review Board (TRB) as a part of their next steps?" field (/governance-review-team/:systemIntakeId/lcid)
- Business Case PDF export button
  - Go to /business/b59fb871-3dfc-47f7-b429-f654f1167580/review
  - Check that "Download Business Case as PDF" button is aligned correctly within grid



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
